tests/automation/landing_page.spec.ts

```diff
@@ -12,17 +12,18 @@ test_Alice_2W(
       waitForElement(aliceWindow1, 'class', 'session-conversation'),
       waitForElement(aliceWindow2, 'class', 'session-conversation'),
     ]);
-    const results = await Promise.allSettled([
-      compareScreenshot(landingPage, `${testInfo.title}`, 'new-account', os),
-      compareScreenshot(
-        restoredPage,
-        `${testInfo.title}`,
-        'restored-account',
-        os,
-      ),
-    ]);
-    if (results.some((r) => r.status === 'rejected')) {
-      throw new Error('One or both screenshot comparisons failed');
-    }
+
+    await compareScreenshot(
+      landingPage,
+      `${testInfo.title}`,
+      'new-account',
+      os,
+    );
+    await compareScreenshot(
+      restoredPage,
+      `${testInfo.title}`,
+      'restored-account',
+      os,
+    );
```

Comment:
this helps by printing exactly what failed with the filepath this is about, see here 
![image](https://github.com/user-attachments/assets/5f5dc159-00b4-47a2-8f5c-b74ee13b04ef)